server/apiRouter.js

```diff
@@ -0,0 +1,11 @@
+const apiRouter = require('express').Router();
+
+const photos = require('../db/controller.js');
+
+apiRouter.get('/rooms/:roomId/', photos.get);
+apiRouter.post('/rooms/:roomId/', photos.post);
+apiRouter.put('/rooms/:roomId/', photos.put);
+apiRouter.patch('/rooms/:roomId/', photos.patch);
+apiRouter.delete('/rooms/:roomId/', photos.delete);
```

Comment:
The entity that these endpoints handle in our REST API is called `Photo`, but these endpoints say nothing about that entity.

This is a violation of the REST mentality at its most fundamental level; Endpoints should end in a noun that refers to the entity we are operating on.

Please add `photos` to the end of each endpoint string.